File: loianeTraining/javaBasico/exercicios-aulas-16-17/Ex13.java
import java.util.Scanner;

public class Ex13 {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        System.out.println("Digite a base e o expoente");
        var base = scan.nextInt();
        var exponent = scan.nextInt();

        var result = base;
        for (int i = 1; i < exponent; i++) {
            result *= base;
        }

        System.out.println(result);
        scan.close();
    }
}

File: loiane-training-csharp/procedural-poo/d/Ex13.cs
using System;

class Ex13
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite a base e o expoente");
        double _base = double.Parse(Console.ReadLine());
        double expoente = double.Parse(Console.ReadLine());

        double sum = 1;
        for (int i = 0; i < expoente; i++)
        {
            sum *= _base;
        }
        Console.WriteLine("Resultado: " + sum);
    }
}